project/Build.scala

```diff
@@ -732,6 +747,7 @@ object Build {
 
   lazy val jsEnvs: Project = (project in file("js-envs")).settings(
       commonSettings,
+      freezeVersionForToolsSettings,
```

Comment:
Why is this necessary for `jsEnvs` and all the projects below it?

Reply:
Because they depend on `logger` which has this setting. So if we do not give them the setting, versions are inconsistent and `update` can (and does) fail in some CI tasks.